### ISSUE TITLE
about ckpt test

### ISSUE THREAD

**Issue by zjz5250** [auroua/InsightFace_TF#68]
@auroua 
     hi：
       the  tricks as follows：
       in eval_data_reader.py
         “if flip == 1:
                img = np.fliplr(img) ”
      and  in  verification.py
       "embeddings = embeddings_list[0] + embeddings_list[1]
        embeddings = sklearn.preprocessing.normalize(embeddings) "
  
       so , why do you  sum  the  embedding  of  one img  and  its flip's?  
  can i use  the embedding  of  one img  directly?
    thanks!

**Comment by JessieW-eyevio** [comment]
i have the same questions
